packages/react-storage/src/components/StorageBrowser/Views/LocationDetailView/LocationDetailView.tsx

```diff
@@ -18,7 +21,28 @@ export const LocationDetailView: LocationDetailView = () => {
 
   const listItems = !hasItems
     ? null
-    : data.items.map(({ key }) => <p key={key}>{key}</p>);
+    : data.items.map((item) => {
```

Comment:
Destructuring simplifies reading `code` and writing boilerplate and helps React with handling reference equality:
```
data.items.map(({ key }) => {
```